src/types.ts

```diff
@@ -328,3 +328,7 @@ export enum Key {
 export interface CheckboxHandles {
   focus(): void;
 }
+
+export interface BannerHandles {
+  focus(): void;
+}
```

Comment:
Maybe CheckboxHandles and BannerHandles should be a single type? Not sure what other kinds of methods we’ll expose on components.

Reply:
I think it's fine to have two interfaces that happen to have the same shape but serve different purposes - if we ever need extra handles on a CheckBox but not a Banner we'd end up having to refactor back into two so I think merging them is premature.